app/lib/actions/katello/host/attach_subscriptions.rb

```diff
@@ -8,12 +8,16 @@ def queue
 
         def plan(host, pools_with_quantities_params)
           action_subject(host)
+
+          subscription_facet = host.subscription_facet
+          fail _("Subscription cannot be attached to the host because of no subscription facet.") unless subscription_facet
```

Comment:
The BZ explains how this action was called through the bulk subscription attach API. Because of that I think we should include the name of the host in the error message, so the user knows which one is unregistered.

`_("Please register host '%s' before attaching subscriptions") % host.name`

Reply:
Wouldn't that be clear on the dynflow console? May be useful in hammer though

Reply:
Would it? I think the dynflow console is extremely hit or miss with the information it contains. Either way, this is the kind of info that should come out of the API. It's a bad experience to have to dig into dynflow to find out what went wrong and the best thing we can do is make users as unaware of dynflow as is possible.